src/teleop/gui/src/components/MoteusStateTable.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+<div class="wrap">
+    <div>
+        <h3> Moteus States </h3>
+    </div>
+    <table class="tableFormat" style="undefined;table-layout: fixed; width: 400px">
+        <colgroup>
+            <col style="width: 20px">
+            <col style="width: 30px">
+            <col style="width: 30px">
+
+        </colgroup>
+
+        <thead>
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Name</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[FrontLeft]}}</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[FrontRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">State</th>
+                <th class="tableElement">{{MoteusStateData.state[FrontLeft]}}</th>
+                <th class="tableElement">{{MoteusStateData.state[FrontRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Error</th>
+                <th class="tableElement">{{MoteusStateData.error[FrontLeft]}}</th>
+                <th class="tableElement">{{MoteusStateData.error[FrontRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Name</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[MiddleLeft]}}</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[MiddleRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">State</th>
+                <th class="tableElement">{{MoteusStateData.state[MiddleLeft]}}</th>
+                <th class="tableElement">{{MoteusStateData.state[MiddleRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Error</th>
+                <th class="tableElement">{{MoteusStateData.error[MiddleLeft]}}</th>
+                <th class="tableElement">{{MoteusStateData.error[MiddleRight]}}</th>
+            </tr>
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Name</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[BackLeft]}}</th>
+                <th class="tableElement" style="font-weight:bold">{{MoteusStateData.name[BackRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">State</th>
+                <th class="tableElement">{{MoteusStateData.state[BackLeft]}}</th>
+                <th class="tableElement">{{MoteusStateData.state[BackRight]}}</th>
+            </tr>
+
+            <tr class="Bold">
+                <th class="tableElement" style="font-weight:bold">Error</th>
```

Comment:
Could we put the font-weight into a CSS class called tableHeader or something? Even if it only has font-weight in it for now it's nice for if we want to edit the styling later.